grr/server/setup.py

```diff
@@ -185,6 +185,8 @@ def make_release_tree(self, base_dir, files):
             "grr_response_server.distro_entry:Worker",
             "grr_admin_ui = "
             "grr_response_server.distro_entry:AdminUI",
+            "grr_afana = "
```

Comment:
I like the `grrafana` name, but naming the binary as `grr_afana` feels somewhat confusing. `grrafana` would be much easier, but then we'd break the convention of prefixing all our binary names with `grr_`. @panhania - what's your take on this?

Reply:
Another possibility can be `grr_grrafana` I guess.

Reply:
No clear opinion here, I am fine with all of them. For the sake of consistency, I would probably pick `grr_grrafana`.

Reply:
+1 for grr_grrafana.